examples/seq2seq/test_seq2seq_examples_multi_gpu.py

```diff
@@ -80,3 +87,31 @@ def convert(k, v):
         self.assertEqual(len(metrics["test"]), 1)
         desired_n_evals = int(args_d["max_epochs"] * (1 / args_d["val_check_interval"]) / 2 + 1)
         self.assertEqual(len(metrics["val"]), desired_n_evals)
+
+    @slow
+    @require_torch_gpu
+    def test_distributed_eval(self):
+        output_dir = self.get_auto_remove_tmp_dir()
+        args = f"""
+            --model_name Helsinki-NLP/opus-mt-en-ro
+            --save_dir {output_dir}
+            --data_dir test_data/wmt_en_ro
```

Comment:
`--task translation` for bleu scores.
`--n_obs 100 --num_beams=2` for (a) faster test, (b) coverage of config pass-through.




(also in your training test PR can use `--eval_beams=2`)

Reply:
> How long does this take? 

As mentioned in the OP - 25sec.

> I can upload a smaller model if that's helpful. (Won't affect coverage.)

We surely can experiment with that.

Reply:
suggestions integrated - thank you! went down from 24secs to 20sec, yay!